include/OpenSpaceToolkit/IO/IP/TCP/HTTP/Client.hpp

```diff
@@ -59,7 +59,7 @@ class Client
     /// @param              [in] aDirectory A destination directory
     /// @return             A file
 
-    static File Fetch(const URL& aUrl, const Directory& aDirectory);
+    static File Fetch(const URL& aUrl, const Directory& aDirectory, const Integer& aFollowCount = 0);
```

Comment:
Maybe make this a `Size` so it can't be negative?

Reply:
Good looks. This is a much better idea.